lib/echo.ex

```diff
@@ -1,18 +1,79 @@
 defmodule Echo do
   @moduledoc """
-  Documentation for `Echo`.
+  This model creates an echo process.
+  You can create multiple echo processes in your application.
+
+  To create an echo process, simply call the start function and pass the pid
+  of the process that will receive responses from the echo process.
+  To interact with this process, you can use the `Echo.ping/1` function or send
+  a message using the `Kernel.send/2`.
+
+  #### Examples
+      iex> {:ok, pid} = Echo.start(self())
+      {:ok, pid}
+      iex> Echo.ping(pid)
+      {:pong, :nonode@nohost}
+
+  If you send an erroneous message, Echo return error tuple `{:error}`.
+
+  #### Examples
+      iex> {:ok, pid} = Echo.start(self())
+      {:ok, pid}
+      iex> send(pid, :some)
+      :some
+      iex> receive do msg -> msg end
+      {:error}
```

Comment:
Tuples of size 1 do not make a lot of sense. 
```suggestion
      :error
```